README.md

```diff
@@ -7,67 +7,98 @@ Perses
 
 ## Overview
 
-Perses is part of the [CoreDash community](https://github.com/coredashio/community).
+Perses is part of the [CoreDash community](https://github.com/coredashio/community). It belongs to the Linux Foundation.
+On a later stage, we want to promote the project to the [Cloud Native Computing Foundation](https://www.cncf.io/) and be
+part of the monitoring tools like Prometheus or Thanos.
+
+Perses is going to tackle multiple different goals:
+
+1. It aims to become a **standard** dashboard visualization tool for Prometheus and other datasources. It will focus on
+   being GitOps-compatible and thus enabling a smooth "dashboards as code" workflow via a new and well-defined dashboard
+   definition model.
+2. While becoming another visualization tool, Perses also aims to provide different npm packages, so it can benefit to
+   anyone that would like to integrate data visualization in his UI. For example, these packages might be used to
+   improve the display of the data in the Prometheus UI.
+3. It also aims to offer a Kubernetes-native mode in which dashboard definitions can be deployed into and read from
+   individual application namespaces (Using CRDs)
+4. To be dashboard as code friendly, we want to provide a complete static validation of the dashboard format. That means
+   you will be able to validate your dashboard in a CI/CD totally statically certainly using
```

Comment:
Probably would get rid of 'totally' and 'certainly'

Reply:
Maybe "completely statically using" instead?

Reply:
I simplified the sentence :). Hope it's better like that !